docs/bin/plugin_formatter.py

```diff
@@ -302,6 +305,28 @@ def get_plugin_info(module_dir, limit_to=None, verbose=False):
         for key, opt in doc.get('options', {}).items():
             doc['options'][key] = normalize_options(opt)
 
+        display.error(module_path)
+        display.error(doc['options'])
+
+        spec, args, kwargs = get_argument_spec(module_path)
+#        try:
+#            spec, args, kwargs = get_argument_spec(module_path)
+#        except AnsibleModuleImportError as e:
+#            display.error('HIER ZIJN WE DAN ALWAAR')
+#            spec = dict()
+#            args = list()
+#            kwargs = dict()
+
+        display.error('BLAH' + str(spec))
```

Comment:
ERROR rather than BLAH :)

Reply:
It is a WIP. There's a very weird issue with Windows modules where the environment gets corrupted with information coming from the previous module. And I cannot pinpoint how this can happen. Part of the logic is skipped in mysterious ways.

Reply:
TBD